### PR TITLE
Make sure FileOutputStream is closed

### PR DESCRIPTION
**Pull request by gverm**
See http://www.playonlinux.org:9000/issues#issues=AVnFemOEqDnz5jcTz9yV

### CODE REVIEW

File: phoenicis-settings/src/main/java/org/phoenicis/settings/SettingsManager.java [hunk 1]
```diff
@@ -78,9 +78,7 @@ public void setRepository(String repository) {
 
     public void save() {
         Settings settings = load();
-        try {
-            File file = new File(settingsFileName);
-            OutputStream outputStream = new FileOutputStream(file);
+        try (OutputStream outputStream = new FileOutputStream(new File(settingsFileName))) {
             DefaultPropertiesPersister persister = new DefaultPropertiesPersister();
             persister.store(settings.getProperties(), outputStream, "Phoenicis User Settings");
         } catch (Exception e) {
```
Comment (qparis) [inline phoenicis-settings/src/main/java/org/phoenicis/settings/SettingsManager.java]: Can you catch a more specific exception here?

Thanks :-)

Comment (gverm) [inline phoenicis-settings/src/main/java/org/phoenicis/settings/SettingsManager.java]: I went for an IOException, is that alright? Thanks for the review :)

**Comment by qparis** [comment]
ok to test

**Comment by qparis** [comment]
IOException needs to be imported

**Comment by gverm** [comment]
My bad, fixed now.

**Pull request merged**